webapp/lib/model.g.dart

```diff
@@ -99,35 +111,43 @@ class Conversation {
     return batch;
   }
 
-  Future<bool> setNotes(DocPubSubUpdate pubSubClient, String newValue) {
+  /// Set notes for the receiver.
```

Comment:
s/receiver/conversation ?

Reply:
Actually I see now that all the other non-static methods use receiver. Can we replace it with the respective object name/type? thanks!

Reply:
Good idea. Fixed.